lms/djangoapps/certificates/tests/factories.py

```diff
@@ -55,7 +55,7 @@ class CertificateHtmlViewConfigurationFactory(DjangoModelFactory):
                 "company_tos_url": "http://www.edx.org/edx-terms-service",
                 "company_verified_certificate_url": "http://www.edx.org/verified-certificate",
                 "document_stylesheet_url_application": "/static/certificates/sass/main-ltr.css",
-                "logo_src": "/static/certificates/images/logo-edx.svg",
+                "logo_src": "/static/certificates/images/logo-edx.png",
```

Comment:
NOTE: had some trouble with .svg files rendering consistently in print across browsers, so I simplified things to use .png graphics.